dev/packages/beats/activemq-0.0.1/kibana/dashboard/26434790-1464-11ea-8fd8-030a13064883.json

```diff
@@ -0,0 +1,33 @@
+{
+    "attributes": {
+        "description": "This dashboard shows application logs collected by the ActiveMQ filebeat module.",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": "{\"filter\":[],\"query\":{\"language\":\"kuery\",\"query\":\"\"}}"
+        },
+        "optionsJSON": "{\"hidePanelTitles\":false,\"useMargins\":true}",
```

Comment:
The JSON strings should be decoded inside the package. I think you have all the logic for it already in place for the encoding, otherwise see https://github.com/elastic/package-registry/pull/214

Reply:
Hmm... if you want to upload them to Kibana, you need to encode them again. I was wondering if it's not better to keep them encoded in packages as well. WDYT?

Reply:
The reason we decode them is for versioning purpose. If you change anything small int he visualisation and then get a PR with the change, it is almost impossible to figure out what change. That is the reason we switched to the decoded format in Beats some while ago and I would like also to use it here. My hope is that in the future Kibana directly supports the decoded option.

The encoding currently happens in the build step.

Reply:
Ah, you're right! Versioning is a killer argument in this thread.

I've just pushed a change that decodes fields.